packages/jellyfish-api-core/__tests__/category/account/listAccountHistory.test.ts

```diff
@@ -199,6 +199,120 @@ describe('Account', () => {
       expect(accountHistories.length).toStrictEqual(1)
     })
   })
+
+  it('should listAccountHistory with options owner mine block height and txn', async () => {
+    const options = {
+      maxBlockHeight: 102,
+      txn: 2
+    }
+    const accountHistories = await client.account.listAccountHistory('mine', options)
+    expect(accountHistories.length).toBeGreaterThan(0)
+    for (let i = 0; i < accountHistories.length; i += 1) {
+      const accountHistory = accountHistories[i]
+      expect(accountHistory.blockHeight).toBeLessThanOrEqual(options.maxBlockHeight)
+      if (accountHistory.blockHeight === options.maxBlockHeight) {
+        expect(accountHistory.txn).toBeLessThanOrEqual(options.txn)
+      }
+    }
```

Comment:
may be we can change to 
```suggestion
    accountHistories.forEach( accountHistory => {
      expect(accountHistory.blockHeight).toBeLessThanOrEqual(options.maxBlockHeight)
      if (accountHistory.blockHeight === options.maxBlockHeight) {
        expect(accountHistory.txn).toBeLessThanOrEqual(options.txn)
      }
    })
```